index.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+

```

Comment:
Kindly remove the extra blank lines and indentations for a cleaner code file.
Please keep in mind that we're only adding blank lines to separate two logical pieces of code or two large sections apart.

Reply:
- [ ] You could include why is this important. 🦊